API/CopilotService/__init__.py

```diff
@@ -55,14 +56,14 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
 
         if service_type == ServiceType.GPT_GENERATION:
             gpt_result = gpt_generate(system_msg, question, history)
-            result = [_build_scenario_response(gpt_result)] if gpt_result else []
+            result = [correct_scenario(_build_scenario_response(gpt_result))] if gpt_result else []
     
         elif service_type == ServiceType.MIX:
             result = knowledge_search(question, top_num)
 
             if len(result) == 0 or not pass_verification(question, result):
                 gpt_result = gpt_generate(system_msg, question, history)
-                result = [_build_scenario_response(gpt_result)] if gpt_result else []
+                result = [correct_scenario(_build_scenario_response(gpt_result))] if gpt_result else []
```

Comment:
In fact, there is also a possibility of data expiration in our knowledge base. How about adding the `correct_scenario()` to the semantic search results of knowledge base? This can maintain consistent processing of results from multiple different sources.

Reply:
Done